usaspending_api/api_contracts/contracts/v2/download/count.md

```diff
@@ -19,6 +19,12 @@ Returns the number of transactions that would be included in a download request
 
     + Attributes (object)
         + `filters` (required, AdvancedFilterObject)
+        + `spending_level` (optional, enum[string])
+            + Members
+                + `subawards`
+                + `transactions`
+                + `awards`
+                + Default: `transactions`
```

Comment:
The indentation is off; `Default` should be at the same level as the `Members` key.
![Screenshot 2024-10-02 091929](https://github.com/user-attachments/assets/caa4a8ac-31f9-4839-a892-05699804a1a3)